Implement Conjugate Views

angrezichatterbox opened the pull request:
<!---
Thank you for your pull request! 🚀
-->

### Contributor checklist

<!-- Please replace the empty checkboxes [] below with checked ones [x] accordingly. -->

-   [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
-   [x] I have tested my code with the `./gradlew lintKotlin detekt test` command as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-Android/blob/main/CONTRIBUTING.md#testing)

---

### Description

<!--
Describe briefly what your pull request proposes to change. Especially if you have more than one commit, it is helpful to give a summary of what your contribution is trying to solve.

Also, please describe shortly how you tested that your change actually works.
-->

This PR bring the conjugate views. I have implemented the conjugate views and tested them individual whether they look good on the keyboard in accordance with heigh and width.For now the conjugate button shows up the 3x2 conjugation view and the display left and right key don't do anything. I have not disabled the key actions for now.

### Related issue

<!--- Scribe-Android prefers that pull requests be related to already open issues. -->
<!--- If applicable, please link to the issue by replacing ISSUE_NUMBER with the appropriate number below. -->
<!--- Feel free to delete this section if this does not apply. -->

-   #267

andrewtavis commented:
Really amazing work, @angrezichatterbox! Thanks so much for taking this on 🙏🙏🙏😊 One thing I'm seeing is that there are two mini buttons on the left hand side of the 3x2 view. Do you know what's up with these? See image below:

![Screenshot_20250418_101708](https://github.com/user-attachments/assets/390cb9d5-f053-4463-b621-01c65f2806db)

angrezichatterbox commented:
I have fixed it. I had added 2 buttons for keeping the alignment correct. But forget to remove it for light mode. I have also fixed the arrows for the dark and light mode.

andrewtavis reviewed (approved):
Absolutely amazing work here, @angrezichatterbox, and thanks so much for the lightning quick response! ⚡ So great to have this finalized so we can make progress on conjugation 😊
